packages/flex-dev-utils/src/packages/lib/__tests__/getRegistryVersion.test.ts

```diff
@@ -0,0 +1,22 @@
+import getRegistryVersion from '../getRegistryVersion';
+
+jest.mock('package-json');
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
+const packageJson = require('package-json');
+
+describe('getRegistryVersion', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  it('should call to package-json', async () => {
```

Comment:
Please add a test for both default (i.e. no tag `getRegistryVersion('foo')`) as well as one with `latest` and one with `beta`